api/rest/config.go

```diff
@@ -209,23 +213,9 @@ func (cfg *Config) loadHTTPOptions(jcfg *jsonConfig) error {
 	key := jcfg.SSLKeyFile
 	cfg.pathSSLCertFile = cert
 	cfg.pathSSLKeyFile = key
```

Comment:
looks like these lines can be deleted as `cfg.tlsOptions()` sets `cfg.pathSSLCertFile` and `cfg.pathSSLKeyFile`

Reply:
fixed